client/components/migration/feature-announcement.jsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Flex, FlexItem, Modal, Icon, Button } from '@wordpress/components';
+import { useCallback, useState } from '@wordpress/element';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import bg from './images/wcshipping-migration.jpg';
+import { Dashboard, Preformatted, LessonPlan, Shipping } from './icons';
+import {
+	isEligableToMigrate,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const FeatureAnnouncement = ({ translate, isEligable }) => {
+	const [isOpen, setIsOpen] = useState(isEligable);
+	const [isUpdating, setIsUpdating] = useState(false);
+
+	const closeModal = useCallback(() => {
+		setIsOpen(false);
+	});
+
+	const snooze = useCallback(() => {
+		// Todo: implement maybe later
+	});
+
+	const update = useCallback(() => {
+		// Todo: implement update
+		setIsUpdating(true);
+		setTimeout(() => setIsUpdating(false), 2000);
+	});
+
+	return <>{isOpen && (<Modal
+		className="migration__announcement-modal"
+		shouldCloseOnEsc={false}
```

Comment:
❔ – Curiosity – “Good to go, but I’d like to learn more.”

Should pressing `esc` not close this modal?

Reply:
We want the users to migrate, I'd rather remove one more way of skipping the migration, but I'm happy to change this if you think we'd better allow them to use escape.
Beside using escape when 2 modals are open may get confusing.

Reply:
Then they can't use the plugin's normal functionality, it will be a hard force! removing the **escape** button functionality is only removing one way of skipping the upgrade while still allowing them to close it if they want. We can ask for opinions from @MatthiasReinholz on this, or we can keep it as is for now and collect their feedback when they test it, I'm not convinced this a big problem.

Reply:
I agree that the `shouldCloseOnEsc` and `shouldCloseOnClickOutside` questions do not block this PR. +1 to discuss it and implement afterwards if needed.